packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx

```diff
@@ -3,7 +3,7 @@ import './awards_section.css';
 
 export const AwardsSection = () => {
     return (
-        <div className="awards-section-hack-aubg-container">
```

Comment:
Changes in the awards section shouldn't be here. You might have created a branch from the previous branch you have been working on. To avoid merge conflicts, please remove these changes made within the AwardsSection.jsx and awards_section.css files